canonical-web-and-design/canonical.com
# 1586 add jobs feed for all roles

## Done
- Add country list to the `send your CV` tab on `/careers/tech-ops` page
- Add job feed to all `/careers` pages

## QA
- Go to [/careers/tech-ops](http://0.0.0.0:8002/careers/tech-ops) and make sure the country select has all the countries
- Go to [/careers/tech-ops](http://0.0.0.0:8002/careers/tech-ops) and click on available roles
- Make sure you get a number of roles in the feed
- Click on one job and check if you can see job details

P.S. I need to refactor the templates used, but that will happen in a separate PR

## Issue

https://github.com/canonical-web-and-design/web-squad/issues/1586
